### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN035.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#82] [General]
# https://blog.3place.cn/3place/lmyn/LMYN035.html

https://blog.3place.cn/3place/lmyn/LMYN035.html

**Comment by picklecai** [comment]
他们看到了恒河，
河中魁首牟尼所住。 （1.34.6）

前面出现了牟尼中魁首，城中魁首，这里又有了河中魁首。

**Comment by picklecai** [comment]
这苦行女立下大誓，
下决心要实行苦行。 （1.34.18）
她实行最严厉的苦行，
她为世人所礼拜讴歌。

在这个故事里，立志苦行的，都是值得尊敬的。
